techniques/failures/F98.html

```diff
@@ -160,26 +156,26 @@ <h3>Example 1: Only using touch-specific JavaScript event listeners when a touch
 }</code></pre>
 			
 				<div class="note">
-				<p class="note-title marker">Note</p>
-				<p>Generally, these approaches will <em>also</em> result in a failure of Success Criterion 2.1.1 Keyboard - but only in situations where a touchscreen interface was also detected.</p>
+				<p class="note-title marker">注記</p>
+				<p>一般的に、これらのアプローチは、達成基準 2.1.1 キーボードの失敗<em>も併せて</em>もたらすが、それはタッチスクリーンインタフェースも検知された状況でのみ、該当する。</p>
 			</div>
 
 			</section>
 				
       <section class="example" id="example-2-hiding/omitting-controls-for-mouse-and-keyboard-users-when-a-touchscreen-is-detected">
-				<h3>Example 2: Hiding/omitting controls for mouse and keyboard users when a touchscreen is detected</h3>
-				<p>Web content containing interactive widgets such as content carousels, with visible buttons to operate the widget (such as previous/next buttons, or a visible scrollbar/slider). These visible controls are hidden/omitted when a touchscreen is detected, under the assumption that users will simply use touch gestures to operate the widgets, and no other alternatives are then provided for keyboard or mouse users.</p>
+				<h3>事例 2: タッチスクリーンが検知されたときに、マウス及びキーボードの利用者のためのコントロールを、非表示／省略している</h3>
+				<p>コンテンツのカルーセルなど、インタラクティブなウィジェットを含むウェブコンテンツで、ウィジェットを操作するための可視ボタン (前へ／次へボタンや、可視スクロールバー／スライダーなど) を持つものがある。これらの可視コントロールは、利用者が単にタッチジェスチャを用いてウィジェットを操作するという仮定のもと、タッチスクリーンが検知されたときに非表示／省略され、キーボード又はマウスの利用者には他の代替手段が提供されない。</p>
 <pre xml:space="preserve"><code class="language-css">/* using CSS Media Queries 4 Interaction Media Features
    to hide particular elements in the page (such as a container
    with visible controls) when a "coarse" primary input is present */
 
 @media (pointer: coarse) {
   #widget .controls { display: none; }
 }</code></pre>
-				<p>Depending on the specific implementation, authors may allow mouse interactions with widgets that mirror touch gestures - for instance, allowing mouse users to also drag/swipe carousels, rather than just relying on clickable previous/next controls or scrollbars. In these cases, hiding controls when a touchscreen is detected will still allow users to operate the widget with the mouse (unless this interaction has also been suppressed/omitted when the touchscreen was detected, as per the previous example). However, if the only keyboard-operable controls for the widget were hidden, and no alternative for keyboard users was provided (such as allowing cursor key operation), this situation would still fail Success Criterion 2.5.6.</p>
+				<p>特定の実装によっては、タッチジェスチャを反映したウィジェットに対して、コンテンツ制作者はマウスによるインタラクションを可能にすることができる。例えば、マウス利用者が、クリック可能な前へ／次へのコントロール又はスクロールバーだけでなく、カルーセルのドラッグ／スワイプもできるようにするなどである。このような場合、タッチスクリーンが検知されたときにコントロールを非表示しても、利用者はマウスでウィジェットを操作することができる (タッチスクリーンが検知されたときに、前掲の例のようにこのインタラクションが抑制／省略されていない限りは)。しかし、そのウィジェットに対してキーボードでのみ操作可能なコントロールが隠されていて、かつキーボード利用者のための代替手段 (カーソルキー操作を可能にするなど) が提供されていない場合、この状況は達成基準 2.5.6 を満たさないことになる</p>
```

Comment:
文末の句点 (。) が抜けていたので追記しました。
```suggestion
				<p>特定の実装によっては、タッチジェスチャを反映したウィジェットに対して、コンテンツ制作者はマウスによるインタラクションを可能にすることができる。例えば、マウス利用者が、クリック可能な前へ／次へのコントロール又はスクロールバーだけでなく、カルーセルのドラッグ／スワイプもできるようにするなどである。このような場合、タッチスクリーンが検知されたときにコントロールを非表示しても、利用者はマウスでウィジェットを操作することができる (タッチスクリーンが検知されたときに、前掲の例のようにこのインタラクションが抑制／省略されていない限りは)。しかし、そのウィジェットに対してキーボードでのみ操作可能なコントロールが隠されていて、かつキーボード利用者のための代替手段 (カーソルキー操作を可能にするなど) が提供されていない場合、この状況は達成基準 2.5.6 を満たさないことになる。</p>
```